src/System.Linq.Queryable/src/Resources/Strings.resx

```diff
@@ -135,4 +135,7 @@
   <data name="NoMethodOnTypeMatchingArguments" xml:space="preserve">
     <value>There is no method '{0}' on type '{1}' that matches the specified arguments</value>
   </data>
+  <data name="EnumeratingNullEnumerableExpression" xml:space="preserve">
+    <value>Cannot enumerate a query created from a null IEnumerable&lt;&gt;</value>
+  </data>
 </root>
```

Comment:
Yes it is tool's fault. I think it is always better to have LF, for example on linux mono develop doesn't make this mistake. :)